### PR TITLE
Fix Mobility Devices Input

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
## Description

This PR fixes the behavior of the checkboxes in the pane for selecting mobility devices:
- "No assistive device" is checked if no other device is selected.
- "No assistive device" gets unchecked if user checks other devices.
- If user checks "No assistive device", other devices that were checked are unchecked.

## PR Checklist
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by amy-corson-ibigroup — APPROVED** [review]
Makes sense and works well! 

The only thing that occurs to me is if mobility devices is set to "none" because the user neglected to choose any options, then I think next time they open the mobility editor, "none" should be checked. But that's a very small note, I'm happy to revisit it down the line.

**Comment by binh-dam-ibigroup** [comment]
> The only thing that occurs to me is if mobility devices is set to "none" because the user neglected to choose any options, then I think next time they open the mobility editor, "none" should be checked.

Oh, it is not?

**Comment by amy-corson-ibigroup** [comment]
Yeah, it looks like if a user goes through the sign up process and doesn't select anything on the mobility devices screen, the box isn't checked when they go into the editor. It does show up on the account page as "Mobility devices: No assistive device"

**Comment by binh-dam-ibigroup** [comment]
> Yeah, it looks like if a user goes through the sign up process and doesn't select anything on the mobility devices screen, the box isn't checked when they go into the editor. It does show up on the account page as "Mobility devices: No assistive device"

Good catch! I made the change in https://github.com/opentripplanner/otp-react-redux/pull/1141/commits/f118e73b8dafb77b71f90c74c4131f2a0aaf7115 so a default is set for all new accounts when `mobilityProfile` is enabled in config.

File: lib/util/user.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ import { isBlank } from './ui'
 
 const { toSentenceCase } = coreUtils.itinerary
 
+export const noneSingleton = ['none']
```
Comment (miles-grant-ibigroup) [inline lib/util/user.js:15]: should this be an all caps const?

**Review by miles-grant-ibigroup — APPROVED** [review]
One tiny nitpick. Overall very clean!

**Pull request merged**